submissions/dashakim/memory-game/script.js

```diff
@@ -44,7 +44,8 @@ let firstCard, secondCard;
 
 const flipCard = e => {
   if (lockBoard) return;
-  const target = e.target.parentElement;
+  const target = e.target;
+  if(!section === target) return;
```

Comment:
That check didn't protect your game from adding `flip` class to section element.

Also, since we need a final tiny change, I would like to meet you with a [destructuring](https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Operators/%D0%94%D0%B5%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D1%83%D1%80%D0%B8%D0%B7%D0%B0%D1%86%D1%96%D1%8F), which came with ES6. Writing function parameter this way `const flipCard = {target} => {...}` gives you possibility to use `target` without declaring `const target = e.target`. Hope you enjoy it :)

Reply:
Yes, I know about destructuring and even use in in `resetBoard` function, but still not everywhere, good advice!